Harvest.Web/ClientApp/src/Tickets/TicketContainer.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect, useState } from "react";
+import { useHistory, useParams } from "react-router-dom";
+import { Project, CreateTicket } from "../types";
+import { ProjectHeader } from "../Requests/ProjectHeader";
+import DatePicker from "react-date-picker";
+import { Button, FormGroup, Input, Label } from "reactstrap";
+
+interface RouteParams {
+    projectId?: string;
+}
+
+export const TicketContainer = () => {
+    const { projectId } = useParams<RouteParams>();
+    const [project, setProject] = useState<Project | undefined>();
+    const [createTicket, setCreateTicket] = useState<CreateTicket>({
+        requirements: "",
+        name: ""
+} as CreateTicket);
+    const [disabled, setDisabled] = useState<boolean>(true);
+    const history = useHistory();
+
+    useEffect(() => {
+        const cb = async () => {
+            const response = await fetch(`/Project/Get/${projectId}`);
+
+            if (response.ok) {
+                const proj: Project = await response.json();
+                setProject(proj);
+                setCreateTicket({...createTicket, projectId:proj.id});
+            }
+        };
+
+        cb();
+    }, [projectId]);
+
+    if (project === undefined) {
+        return <div>Project Not Found!</div>;
```

Comment:
project will be undefined if the fetch hasn't come back yet too, so it's not always because project not found.